sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledDataAsync.java

```diff
@@ -79,21 +89,36 @@ public static void main(String[] args) throws IOException {
                 }
             });
 
-        //  The main difference is found in the labels of its fields
-        // The form recognized with a labeled model will have the labels it was trained with,
-        // the unlabeled one will be denoted with indices
         System.out.println("--------Recognizing forms with labeled custom model--------");
+        // With a form recognized by a model trained with labels, the `formField.getName()` key will be its label
+        // given during
+        // training
+        // `value` will contain the typed field value and `valueText` will contain information about the field value
```

Comment:
nit: weird formatting here

Reply:
Could be the diff mode, https://github.com/Azure/azure-sdk-for-java/blob/4364c78509af8849ca38fed9c030d9608ad5e13b/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledDataAsync.java#L94

Reply:
Weird! Okay ignore me :)